src/components/ActionControls/ActionControls.js

```diff
@@ -124,6 +124,10 @@ class ActionControls extends React.Component<Props, State> {
                     />
                 );
             case TYPES.draw:
+                if (!canDelete) {
```

Comment:
I'm curious about this logic. How does deletion affect creation, and how is it only for certain kinds of annotation types, and not all? (also in the above `Types.highlight`)

Reply:
If you don't have delete permissions on an annotation you can still comment so long as you have the `can_annotate` permission. 

If you're creating the annotaiton, by default canDelete is true

Reply:
I can add a note in ActionControls.js as well